backend-worker/worker.py

```diff
@@ -1,26 +1,546 @@
-        time.sleep(1)
+        response = requests.get(job_queue_url)
+        if not response.json:
+            time.sleep(POLLING_INTERVAL_S)
+            continue
+
+        server_args = response.json['server-attrs']
+        job_type = response.json['job_type']
+        job_args = response.json['job_args']
+
+        try:
+            if job_type in {JOBTYPE_FILL, JOBTYPE_WALLPAPER}:
+                if job_type == 'parcel_fill':
+                    fill_workspace = tempfile.mkdtemp(
+                        prefix='fill-', dir=local_appdata_dir)
+                    result_path = f'{fill_workspace}/filled.tif'
+                    fill_parcel(
+                        parcel_wkt_epsg3857=job_args['target_parcel_wkt'],
+                        fill_lulc_class=job_args['lulc_class'],
+                        source_nlud_raster_path=job_args['lulc_source_url'],
+                        target_lulc_path=result_path
+                    )
+                elif job_type == 'wallpaper':
+                    wallpaper_workspace = tempfile.mkdtemp(
+                        prefix='wallpaper-', dir=local_appdata_dir)
+                    result_path = f'{wallpaper_workspace}/wallpaper.tif'
+                    wallpaper_parcel(
+                        parcel_wkt_epsg3857=job_args['target_parcel_wkt'],
+                        pattern_wkt_epsg3857=job_args['pattern_bbox_wkt'],
+                        #source_nlud_raster_path=f'{local_appdata_dir}/nlud.tif',
+                        source_nlud_raster_path=job_args['lulc_source_url'],
+                        target_raster_path=result_path,
+                        working_dir=wallpaper_workspace
+                    )
+                data = {
+                    'result': {
+                        'lulc_path': result_path,
+                        'lulc_stats': {
+                            'base': pixelpercents_under_parcel(
+                                job_args['target_parcel_wkt'],
+                                job_args['lulc_source_url']
+                            ),
+                            'result': pixelpercents_under_parcel(
+                                job_args['target_parcel_wkt'],
+                                result_path
+                            ),
+                        }
+                    },
+                }
+            elif job_type == 'stats_under_parcel':
+                data = {
+                    'result': {
+                        'lulc_stats': {
+                            'base': pixelpercents_under_parcel(
+                                job_args['target_parcel_wkt'],
+                                job_args['lulc_source_url']
+                            ),
+                        }
+                    }
+                }
+            else:
+                raise ValueError(f"Invalid job type: {job_type}")
+            status = STATUS_SUCCESS
+        except Exception as error:
```

Comment:
Ah, so if one of those above `asserts` fails we'll update the status here. Perfect.